src/web/app/page.component.html

```diff
@@ -45,7 +45,7 @@
 <tm-loader-bar></tm-loader-bar>
 <tm-toast [(toast)]="toast" aria-live="polite" aria-atomic="true"></tm-toast>
 <main id="main-content" class="container main-content">
-  <tm-notification-banner *ngIf="isStudent || isInstructor" [url]="getUrl()" [notificationTargetUser]="notificationTargetUser"></tm-notification-banner>
+ <tm-notification-banner *ngIf="isStudent || isInstructor" [url]="getUrl()" [notificationTargetUser]="notificationTargetUser"></tm-notification-banner>
```

Comment:
Just a nit: there should be a spacing i.e., 2 spaces, so that it is in the same column as the code in L49.

For clarity (dashes represent spaces):
```html
--<tag1>
--<tag2>
```

Reply:
I made a modification for this.
The E2E test fail again; and the failed component-testing work well in my local environment. 
Please let me know anything else I can do for help.
Thanks!

Reply:
No worries, I will re-run the test.